### PR TITLE
user misconfiguration does not need to be reported to airbrake

### PR DESCRIPTION
**Pull request by grosser**
previous errors were mor of the mysql is broken / the app is failing kind,
but this is simply user error and does not need to be reported

@zendesk/runway 

### Risks
 - None

### CODE REVIEW

File: config/database.mysql.yml.example [hunk 1]
```diff
@@ -1,4 +1,7 @@
-<% uri = URI(ENV['MYSQL_URL'] || 'mysql://root@127.0.0.1:3306/samson_development') %>
+<%
+  uri = URI(ENV['MYSQL_URL'] || 'mysql://root@127.0.0.1:3306/samson_development')
```
Comment (grosser) [inline config/database.mysql.yml.example]: @gabetax fyi this needed mysql2

Comment (grosser) [inline config/database.mysql.yml.example]: leaving the mysql on purpose to only have 1 code-path

File: lib/samson/hooks.rb [hunk 2]
```diff
@@ -1,5 +1,8 @@
 module Samson
   module Hooks
+    class UserError < StandardError
```
Comment (grosser) [inline lib/samson/hooks.rb]: this is a weird place to add the error ... but idk where else to add ...

**Comment by jonmoter** [comment]
Seems reasonable to me.  :+1:

File: config/database.mysql.yml.example (hunk 1 above)
Comment (gabetax) [inline config/database.mysql.yml.example]: I made the same change in my other repo. good call.

**Pull request merged**